pom.xml

```diff
@@ -591,6 +600,16 @@
                 <artifactId>h2</artifactId>
                 <version>${h2.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.hibernate.validator</groupId>
+                <artifactId>hibernate-validator</artifactId>
+                <version>${hibernate-validator.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.glassfish.expressly</groupId>
+                <artifactId>expressly</artifactId>
+                <version>${expressly.version}</version>
+            </dependency>
```

Comment:
This dependency needs approval from the CNCF since it's EPL 2.0. We can merge without approval from the CNCF, but would need to get an exception at some point, ideally before the release.